### ISSUE TITLE
How to enable progress bar preview ?

### ISSUE THREAD

**Issue by pawelwiejkut** [vimeo/player.js#782]
### Expected Behavior

Display progress bar preview like on youtube:

![Screenshot 2021-11-13 at 23 19 46](https://user-images.githubusercontent.com/11619019/141660644-b965051d-649a-450d-a5b6-ef21e86f0dc4.png)

### Actual Behavior

I can't find any way to enable this. Is this function available?

### Steps to Reproduce

Run video and put your mouse on the progress bar (timeline).

**Comment by rkrishnan8594** [comment]
Hi @pawelwiejkut - this feature should be automatically present on all videos uploaded over the past two months. We are in the process of slowly backfilling thumbnail previews on videos older than 2 months. There are no plans at the moment to add functionality to our API for enabling or disabling thumbnail previews, but that is something we will discuss internally.

**Comment by pawelwiejkut** [comment]
Hi @rkrishnan8594, it should work also when it is embedded? To be honest this feature is not working on my videos uploaded even yesterday. I have checked on my site and on Vimeo and in both cases, I can't see the preview.

**Comment by rkrishnan8594** [comment]
Yes, it should work when embedded. It would only be hidden on smaller player sizes. Would you mind sharing where you have your videos embedded? I can forward this to the team if it's a bug.

**Comment by pawelwiejkut** [comment]
Can you please check this video:
https://vimeo.com/manage/videos/645606872/6e5505ef62/

I can't see the preview:
<img width="817" alt="Screenshot 2021-11-15 at 20 15 11" src="https://user-images.githubusercontent.com/11619019/141840597-e8fe3562-b4c6-460a-adc9-23cc9bd37bdb.png">

It is not working for me on Safari, Chrome, or Edge. I even change my internet the connection to the hotspot.

**Comment by rkrishnan8594** [comment]
@pawelwiejkut Can you check now? Should be working. We resumed our rollout of the feature after some technical issues last week.

**Comment by pawelwiejkut** [comment]
@rkrishnan8594  working! on this and other videos too. Thank you very much !!

**Comment by JagadeesanCoastal9714** [comment]
![video_duration](https://github.com/vimeo/player.js/assets/159803024/f4003ca9-eb52-49b2-9fd0-1bfa8988bed1)


It is not possible to display video length in the Vimeo progress bar. Could you please advise me?
